src/Microsoft.SqlTools.ServiceLayer/Formatter/TSqlFormatterService.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.SqlTools.ServiceLayer.Formatter.Contracts;
 using Microsoft.SqlTools.ServiceLayer.Hosting;
 using Microsoft.SqlTools.ServiceLayer.Hosting.Protocol;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices;
```

Comment:
I'm somewhat concerned that a "general purpose" telemetry event sending is being done in the LanguageService. It seems to me that this would cause a lot of tangling between services. If telemetry is something to offer up as a service for other services to consume, we should make it its own service. If there's precedent to have it in the LanguageService, we should decouple it from there and make it its own service.